scripts/nbconverted/generate_input_PA1673like_v1.py

```diff
@@ -0,0 +1,169 @@
+
+# coding: utf-8
+
+# In[1]:
+
+
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# In[2]:
+
+
+#-------------------------------------------------------------------------------------------------------------------------------
+# By Alexandra Lee (Septermber 2018) 
+#
+# Generate input files
+#
+# Dataset: Pseudomonas aeruginosa gene expression compendium referenced in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5069748/
+# 
+# Group samples based on gene expression of PA1673
+#
+# Generate offset vector using extreme gene expression values (train_offset_original):
```

Comment:
unclear what an "offset vector" is? Maybe b/c I am new to the project

Reply:
The offset vector is another word for our "perturbation vector."  This vector represents the difference in gene expression between the extreme points that you will be interpolating between (i.e. chair with arms and chair without arms, disease and normal).  Then we are adding some portion of this "perturbation/offset vector" to our baseline gene expression to interpolate and make predictions about the intermediate gene expression values.  

I need to add more comments to clean up the code which I hope to do when I refactor everything to.  So that the analysis is clear to users